app-emulation/rkt/rkt-0.8.0.ebuild

```diff
@@ -0,0 +1,95 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+AUTOTOOLS_AUTORECONF=yes
+AUTOTOOLS_IN_SOURCE_BUILD=yes
+
+inherit autotools-utils flag-o-matic systemd toolchain-funcs
+
+KEYWORDS=""
```

Comment:
It should be ~amd64 keyworded, but for that we need to mask the rkt_stage_{src,host} use-flags on the following profiles:

```
dependency.bad [fatal]        4
   app-emulation/rkt/rkt-0.8.0.ebuild: DEPEND: ~amd64(hardened/linux/amd64/no-multilib/selinux)
['>=sys-apps/systemd-224']
   app-emulation/rkt/rkt-0.8.0.ebuild: RDEPEND: ~amd64(hardened/linux/amd64/no-multilib/selinux)
['>=sys-apps/systemd-224']
   app-emulation/rkt/rkt-0.8.0.ebuild: DEPEND: ~amd64(hardened/linux/amd64/selinux)
['>=sys-apps/systemd-224']
   app-emulation/rkt/rkt-0.8.0.ebuild: RDEPEND: ~amd64(hardened/linux/amd64/selinux)
['>=sys-apps/systemd-224']
```